recipe/meta.yaml

```
@@ -1,33 +1,29 @@
 {% set name = "prettytable" %}
-{% set version = "0.7.2" %}
-{% set hash_type = "sha256" %}
-{% set hash = "2d5460dc9db74a32bcc8f9f67de68b2c4f4d2f01fa3bd518764c69156d9cacd9" %}
-{% set build = 3 %}
+{% set version = "1.0.1" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  {{ hash_type}}: {{ hash }}
+  sha256: 6bb7f539903cb031fecb855b615cbcac8cd245ebc6fa51c6e23ab3386db89771
 
 build:
   noarch: python
-  number: {{ build }}
-  script: python -m pip install --no-deps --ignore-installed .
-
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - pip
     - python
```

This should match run:
```suggestion
    - python >=3.5
```